quickstart_lists/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains 'starter lists' of packages for bootstrapping and syncing on-prem-builders.
+
+Background: Historically we bootstraped on-prem-builders by downloading all the packages in 'core'
+for all targets. That amounted to about 15GB, and was both too much and too little, in that many of
+the packages weren't needed, and for many patterns (effortless) other origins were needed.
+
+The basic naming pattern is TASK\_ARCH\_CHANNEL. This due to a limitation of the input file
+format. The simple newline separated list of package idents doesn't allow for specification of
+channel or target architecture inline, so we're using a file naming convention to represent that
+information.
+
+The current tasks are
+* builder (setting up an on prem builder)
```

Comment:
Should we also add some estimates for disk space requirements (and anticipated download times) for each task?